docs/mkdocs/documentation/install.md

```diff
@@ -1,6 +1,10 @@
 # Installing
 
-## Using `kubectl`
+## With [OLM](https://olm.operatorframework.io/) (recommended)
+
+Follow the instructions on [OperatorHub.io](https://operatorhub.io/operator/kernel-module-management).
+
+## With `kubectl`
```

Comment:
We should probably link it to https://github.com/kubernetes-sigs/kernel-module-management#getting-started (or vise versa).

Same for hub-spoke docks (instead of duplicating the commands).

Reply:
I think we should rather update `README.md` to point to the docs. We should actually discourage users from installing the bleeding edge version - that can lead to problems. That is why we now recommend that people install from Operator Hub instead of `kubectl`.